nvflare/private/fed/client/client_executor.py

```diff
@@ -336,7 +319,7 @@ def abort_app(self, job_id):
             if process_status == ClientStatus.STARTED:
                 try:
                     with self.lock:
-                        child_process = self.run_processes[job_id][RunProcessKey.CHILD_PROCESS]
+                        job_launcher = self.run_processes[job_id][RunProcessKey.JOB_HANDLE]
```

Comment:
The var name should be "job_handle", not "job_launcher".

Reply:
renamed